sli/tokenutils.cc

```diff
@@ -97,13 +106,20 @@ template <>
 float
 getValue< float >( const Token& t )
```

Comment:
I don't think we use float anywhere in NEST, could you try to remove this function and see if anything breaks?

Reply:
I removed this function from the code and everything seems to work fine, see  c21eef40b26d45b801cfa606d6bdb6917a99d219.